interface/skin.dmf

```diff
@@ -2,735 +2,553 @@ macro "borghotkeymode"
 	elem 
 		name = "TAB"
 		command = ".winset \"mainwindow.macro=borgmacro hotkey_toggle.is-checked=false input.focus=true input.background-color=#D3B5B5\""
-		is-disabled = false

```

Comment:
Nothing, because is-disabled being false is the default.

I didn't actually remove those, Byond did. If you just open skin.dmf in DreamMaker right now and hit Ctrl-S, it will remove all those because it knows they aren't necessary.

I assume they were put there by someone editing skin.dmf in an older version of DreamMaker that didn't know any better and it put them in.